timescaledb/tutorials/grafana/visualizations/time-series.md

```diff
@@ -0,0 +1,244 @@
+
+    Because the stock market is only open from 9:30&nbsp;AM to 4:00&nbsp;PM on 
+    weekdays, there are large gaps in our dataset where there is no data. 
+    Grafana automatically connects the last non-null value to the nearest other 
+    non-null value. This creates the long, straight, almost-horizontal lines you see
+    in the graph.
+    
+1.  To circumvent this issue, you can use Grafana's `Connect null values` settings. But first,
+    you need rows containing null values wherever you have no data. By default,
+    `time_bucket` doesn't return a row if there is no data. In your query, replace `time_bucket`
+    with [`time_bucket_gapfill`](https://docs.timescale.com/api/latest/hyperfunctions/gapfilling-interpolation/time_bucket_gapfill/).
+    If you don't specify a gapfilling function, `time_bucket_gapfill` returns a row with a
+    null value wherever there is no data.
+
+1.  In your query, replace `time_bucket` with `time_bucket_gapfill`.
+   
+    ```SQL
+    SELECT time_bucket_gapfill('$bucket_interval', time) as time,
+	    AVG(price) as price 
+    FROM stocks_real_time
+    WHERE symbol = '$symbol'
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    GROUP BY time_bucket_gapfill('$bucket_interval', time)
+    ORDER BY time;
+    ```
+
+1.  In the options panel, set `Connect null values` to `Threshold`. Give `Threshold` a value
+    of `24h`.
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/time-series/connect-null-values.png" alt="Screenshot of the 'Connect null values' in the time-series options panel. The selected value is 'Threshold' and it has a value of less than 24 hours."/>
+
+1.  Grafana returns a graph similar to this one:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/time-series/time-bucket-gapfill-graph.png" alt="Screenshot of the time-series graph produced by Grafana using the `time_bucket_gapfill` hyperfunction. The graph represents the price of AMD stock in the past 30 days with an interval of 30 minutes and null values for every gap larger than 24 hours"/>
+
+    This graph allows you to better visualize the stock price during the week.
+    It bridges the less-than-24-hour gap between 4:00&nbsp;PM and 9:30&nbsp;AM,
+    but doesn't connect the values over the weekend.
+
+</procedure>
+
+## Create multiple time-series graphs in a single panel
+
+If you want to compare the price of 2 stocks over time, you can make 
+2 separate panels with 2 separate symbol variables. A better alternative is to 
+combine the 2 time-series graphs into a single panel. To do this, 
```

Comment:
```suggestion
If you want to compare the price of two stocks over time, you could make 
two separate panels with separate symbol variables, however, a better alternative is to 
combine the two time-series graphs into a single panel. To do this, 
```
I'm a big fan of using numeric values when it makes sense, but using "two" here is proper. When we refer to things like "one hour" in a `time_bucket` function, I instead push for "1 hour" (or "1-hour") because that's what the literal parameter value is.

Here though, <=10 usually means we write it out.